synapse/cortex.py

```diff
@@ -6354,6 +6354,10 @@ async def editCronJob(self, iden, name, valu):
         mesg = f'editCronJob name {name} is not supported for editing.'
         raise s_exc.BadArg(mesg=mesg)
 
+    @s_nexus.Pusher.onPushAuto('cron:reload')
```

Comment:
This feels like a bit of an anti-pattern. Shouldn't whatever event propagates the state changes be used to trigger the cron job to have an updated state vs waiting for some subsequent "reload" event?

Reply:
Some of this ickyness comes from that fact that is that the persistent state data is currently propagated via the hive already, but not reloaded in the clients.

We could change is such that all the updates to the `appt` objects are done through a dedicated nexus handler and then we would have a single way to update the live object and the state data?

Reply:
That would be way better 👍